libvast/test/fixtures/events.cpp

```diff
@@ -60,15 +64,15 @@ std::vector<event> events::bgpdump_txt;
 std::vector<event> events::random;
 
 std::vector<table_slice_handle> events::bro_conn_log_slices;
-// std::vector<table_slice_handle> events::bro_dns_log_slices;
-// std::vector<table_slice_handle> events::bro_http_log_slices;
-// std::vector<table_slice_handle> events::bgpdump_txt_slices;
+std::vector<table_slice_handle> events::bro_dns_log_slices;
+std::vector<table_slice_handle> events::bro_http_log_slices;
+std::vector<table_slice_handle> events::bgpdump_txt_slices;
 // std::vector<table_slice_handle> events::random_slices;
 
 std::vector<const_table_slice_handle> events::const_bro_conn_log_slices;
 // std::vector<const_table_slice_handle> events::const_bro_http_log_slices;
 // std::vector<const_table_slice_handle> events::const_bro_dns_log_slices;
```

Comment:
I left them commented because they aren't used anywhere. The non-const ones were all activated because id assignment and slicing happens in lockstep now, and it's better to adjust expected ids in the writer tests just once, instead of every time we activate one of these inputs.

I will enable them in the next update.